gazebo_ros/include/gazebo_ros/qos.hpp

```diff
@@ -20,6 +20,8 @@
 #include <rclcpp/node_options.hpp>
 #include <rclcpp/qos.hpp>
 
+#include <gazebo_ros/node_visibility_control.h>
```

Comment:
not related to this PR, but the visibility control file is not following the normal naming conventions neither the macros.

why wasn't the file named `gazebo_ros/visibility_control.h` and the macros `GAZEBO_ROS_PUBLIC`, etc?

Reply:
it was added in #1077; I think the reviewers from the ignition team didn't know about the ros2 naming conventions

Reply:
> it was added in #1077; I think the reviewers from the ignition team didn't know about the ros2 naming conventions

I see that there are two libraries being built in the same package, maybe the idea was to control the visibility of both independently.

Reply:
Yes, the motivation was to define the visibility control headers independently since typically one library requires its own visibility control namespace.